ru/schema-sources/atack1.drawio

```diff
@@ -4,7 +4,7 @@
       <root>
         <mxCell id="0" />
         <mxCell id="1" parent="0" />
-        <mxCell id="2" value="Рука Алисы" style="swimlane;whiteSpace=wrap" parent="1" vertex="1">
+        <mxCell id="2" value="{{ general.hand }} {{ general.first_player }}" style="swimlane;whiteSpace=wrap" parent="1" vertex="1">
```

Comment:
I think we should store that as a single placeholder. `{{ atack.first_player_hand }}`

Reply:
I did this because some information can be shared accross schemas but in this case when can use `{{ general.first_player_hand }}`, the reason for this is both _attack_ and _game process_ share this

Reply:
Right now it is not correct grammatically :)
I propose duplicating the labels.

Reply:
Done!

Reply:
@sobolevn I've change every schema now!